CHANGELOG.md

```diff
@@ -4,8 +4,12 @@
 
 ## [Unreleased]
 
-### Breaking changes
+### New features
+- Add vertical alignement modifiers in `objects.media`. [[#379](https://github.com/inuitcss/inuitcss/issues/379)]
```

Comment:
Consider writing `.o-media` instead of `objects.media`, as 

1. This is how we referenced objects in the changelog in the past
2. This is probably a more common thing people will search for in the changelog